columnphysics/icepack_intfc.F90

```diff
@@ -124,6 +124,8 @@ module icepack_intfc
       use icepack_mushy_physics , only: icepack_mushy_liquid_fraction
       use icepack_mushy_physics , only: icepack_mushy_temperature_mush
 
+      use icepack_meltpond_sealvl, only: icepack_init_sealvlpnd
```

Comment:
This new subroutine should have the special comment tags so we generate interface documentation.  I can do that and generate the new documentation after this PR is merged and before the release if it doesn't get done now.